dockerfiles/build/www/wp-plugins/public.txt

```diff
@@ -0,0 +1,16 @@
+# List of plugins to download from the public WordPress plugins repository.
```

Comment:
I suggest adding a URL to the repository where those plugins can be found, just to make sure everyone has the same reference.